styleguide/css/main.scss

```diff
@@ -1,13 +1,9 @@
-$white: #fff;
-$light-gray-white: #efefef;
-$dark-gray: #333;
-$medium-gray: #666;
-
+@import '../_themes/global/scss/vendor_prefixes';
+@import '../_themes/global/scss/swatch';
```

Comment:
I pulled in the color swatch and vendor prefixes into main.scss just so we don't have to maintain separate lists for these color swatches. Since these includes are mixins/variables only, they should have zero effect on the final contents of main.css